hazelcast/src/main/java/com/hazelcast/client/HazelcastClient.java

```diff
@@ -487,8 +497,4 @@ static String getInstanceName(ClientConfig clientConfig, ClientFailoverConfig fa
         }
         return instanceName;
     }
-
-    static ConcurrentMap<String, InstanceFuture<HazelcastClientProxy>> getClients() {
-        return CLIENTS;
-    }
```

Comment:
Was this public API?  I think HazelcastClient is public.

Reply:
No, this method was package-private